nailgun/entities.py

```diff
@@ -226,6 +226,7 @@ def __init__(self, server_config=None, **kwargs):
                 Organization,
                 required=True,
             ),
+            'organization_id': entity_fields.IntegerField(),
```

Comment:
There's organization field already present, so not sure why organization_id field is required here?

Reply:
Let me take a look at it. Nailgun itself should take care of including `organization_id` in the HTTP request.

Reply:
@ogajduse if you look at this response. 
```Received HTTP 404 response: {"displayMessage":"One of parameters [ organization_id ] required but not specified."```

organization_id is not being passed in. Even when I played around with the existing organization entity. I could not get the ID from it.